Tests/Imagine/Filter/PostProcessor/PngquantPostProcessorTest.php

```diff
@@ -11,25 +11,177 @@
 
 namespace Liip\ImagineBundle\Tests\Imagine\Filter\PostProcessor;
 
-use Liip\ImagineBundle\Binary\BinaryInterface;
 use Liip\ImagineBundle\Imagine\Filter\PostProcessor\PngquantPostProcessor;
 use Liip\ImagineBundle\Model\Binary;
-use Liip\ImagineBundle\Tests\AbstractTest;
+use Liip\ImagineBundle\Model\FileBinary;
 
 /**
+ * @covers \Liip\ImagineBundle\Imagine\Filter\PostProcessor\AbstractPostProcessor
  * @covers \Liip\ImagineBundle\Imagine\Filter\PostProcessor\PngquantPostProcessor
  */
-class PngquantPostProcessorTest extends AbstractTest
+class PngquantPostProcessorTest extends AbstractPostProcessorTestCase
 {
-    public function testPngquantPostProcessor()
+    /**
+     * @group legacy
+     *
+     * @expectedDeprecation The %s::setQuality() method was deprecated in %s and will be removed in %s. You must setup the class state via its __construct() method. You can still pass filter-specific options to the process() method to overwrite behavior.
```

Comment:
i'd assume there is an expectDeprecation method as well on phpunit, but i could be mistaken.